.eslintignore

```diff
@@ -1,13 +1,8 @@
-/examples/*/node_modules
-/examples/*/dist
-/packages/*/node_modules
-/packages/@sanity/*/node_modules
-/packages/*/dist
-/packages/@sanity/*/dist
-/packages/*/lib
-/packages/*/coverage
-/packages/@sanity/*/lib
-/packages/@sanity/*/coverage
+**/dist
+**/node_modules
+**/lib
```

Comment:
note to self: check for `lib` folders in `src` (or reconsider)

Reply:
I sometimes use `lib` directory in source code. We should switch to `dist` for compiled files imo. 😇

Reply:
I know `lib` means something different in other languages/ecosystems, but in the JS world I feel it's so established as being "compiled source" (similar to dist) that I tend to read it as an error when things import from say `../../lib/some/thing`. So would personally steer away from using it.